docs/governance/community-guidelines/releases/cmb.md

```diff
@@ -0,0 +1,106 @@
+# Change Management Board
+
+This document is a [community guideline].
+
+## Purpose
+
+The document outlines and defines the guidelines for the Change Management Board (CMB) for the Common Cloud Controls (CCC) project.
+
+The CMB is a body of representatives from financial institutions of varying sizes and types. Its primary role is to review and approve changes and new catalogs that are within the Release Candidate. The CMB collectively represents end-user stakeholders, ensuring that each artifact is adaptable to the needs of a wide range of institutions while maintaining consistency and integrity across the board.
+
+## Membership
+
+The change management board is comprised of a Release Manager and the body of reviewers.
+
+A release cycle shall be a minimum of one month, during which time a Release Manager will solicit and arbitrate feedback from the reviewers prior to approving and initiating the release.
+
+### Release Manager Responsibilities
+
+The release manager is not a unilateral authority on the release, rather they are the representative of the group's opinions. Insomuch as they represent the CMB, the release manager holds the final guidance in the lifecycle of an asset.
+
+The release manager will be responsible for the following:
+
+- Collaborate with the CCC working group leads to ensure that the asset is ready for review.
+- Issue an announcement to the CMB, containing:
+  - Links to the asset under review
+  - Desired release date
+  - Deadline for initial responses (two weeks prior to desired release date)
+  - Instructions for participating in this review cycle
+- When a change request (CR) is received:
+  - Evaluate the quality of the CR. If necessary, request adjustments for clarity or conciseness.
+  - Relay the CR to all participating reviewers
+  - At least two members must agree on a CR before it moves forward, with majority opinion ruling when there is dissent.
```

Comment:
Ah, I see, this is where the operating policy is defined. Should it be here? Or in the same area as the charter for SteerCo, WG Leads and so on?

Reply:
Good point... I think this is a good location for now though